libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2771,6 +2771,8 @@ void GCS::create_gcs_mavlink_backend(AP_HAL::UARTDriver &uart)
         return;
     }
 
+    // fill in the device ID
+
```

Comment:
It looks like you added the comment but not the code that actually sets the devid?

Reply:
Note the "WIP" label.  People asked on Discord about where this was up to, I decided to show not tell.... nad sak if anyone else wanted to take it up before I came back to it.